openedx/core/djangoapps/waffle_utils/testutils.py

```diff
@@ -0,0 +1,60 @@
+"""
+Test utilities for waffle utilities.
+"""
+
+from functools import wraps
+
+from waffle.testutils import override_flag
+
+
+def override_waffle_flag(flag, active):
```

Comment:
Agreed that we need a test, which I will add in waffle_utils.  Once this is tested, I don't think we'll need that level of control in the decorator, but I could be wrong.